README.md

```diff
@@ -25,7 +25,22 @@ Unlike other observability backends, it has a simple and easy-to-manage architec
 with just two components: the Promscale Connector and the Promscale Database (PostgreSQL with the
 TimescaleDB and Promscale extensions).
 
-Key features:
+## Quick Start
+
+**Try it out now** with our demo environment you can deploy on your laptop in five minutes with Docker.
+
+```bash
+git clone git@github.com:timescale/promscale.git
+cd promscale/docker-compose/
+docker compose up -d
+```
+Explore your metrics and traces in Grafana (http://localhost:3000, username: admin, password: admin) and
```

Comment:
Grafana is not being set up in the docker-compose.yaml . Also, I had an issue with Jaeger, in the docker-compose it doesn't have:

```
    depends_on:
      - promscale
    restart: on-failure
```

So it started before timescale and promscale, it failed and didn't restart so I ended up with a broken deploy and couldn't access the Jaeger UI. I should've catch this in the previous PR 😅

Reply:
I see that @ramonguiu already merged to master the update to the docker-compose with the restarts. Then there's just the issue with the mention of grafana.